apps/server/src/adapterSetup.ts

```diff
@@ -1,22 +1,32 @@
 import { getMxAdapterMapObject } from "@ucp-npm/mx-adapter";
+import { getSophtronAdapterMapObject } from "@ucp-npm/sophtron-adapter";
+import { getTemplateAdapterMapObject } from "@ucp-npm/template-adapter";
+import type { AdapterMap } from "@repo/utils";
+
+import {adapterMapObject as testAdapterMapObject } from "./test-adapter";
 import config from "./config";
 import { get, set } from "./services/storageClient/redis";
 import * as logger from "./infra/logger";
-import { SophtronAdapter } from "./adapters/sophtron";
-import getSophtronVc from "./services/vcAggregators/sophtronVc";
-import { adapterMapObject as testAdapterMapObject } from "./test-adapter";
-import { getTemplateAdapterMapObject } from "@ucp-npm/template-adapter";
 
 const templateAdapterMapObject = getTemplateAdapterMapObject();
 
-const sophtronAdapterMapObject = {
-  sophtron: {
-    vcAdapter: getSophtronVc,
-    widgetAdapter: new SophtronAdapter(),
-  },
-};
+const sophtronAdapterMapObject: Record<string, AdapterMap> =
+  getSophtronAdapterMapObject({
+    logClient: logger,
+    aggregatorCredentials: {
+      clientId: config.SophtronApiUserId,
+      secret: config.SophtronApiUserSecret,
+      endpoint: config.SophtronApiServiceEndpoint,
+      vcEndpoint: config.SophtronVCServiceEndpoint,
+      aggregator: "sophtron",
+      available: true,
```

Comment:
What is available for?

Reply:
I'm sure I just copied these from somewhere else. I didn't add that.